TESTING.md

```diff
@@ -64,6 +64,17 @@ possible torun a subset of the test suite by adding
 mvn test -DexcludedGroups=unlimited-crypto
 ```
 
+## Dry run
+	
+It's possible to print a list of which integration tests would run and their relevant
```

Comment:
This may be better phrased:

> The following invocation will print a list of which integration tests would run and their relevant
parameters:

Reply:
Noted.